src/lesson13/Main.java

```diff
@@ -0,0 +1,66 @@
+package lesson13;
+
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+
+//1
+        String string = "What is Java technology and why do I need it? Java is a programming language and computing platform first released by Sun Microsystems in 1995. It has evolved from humble beginnings to power a large share of today’s digital world, by providing the reliable platform upon which many services and applications are built. New, innovative products and digital services designed for the future continue to rely on Java, as well.";
+        Collection<String> words = new ArrayList<>(List.of(string.split(" ")));
+        System.out.println(words.size()); // number of words = 70;
+
+        System.out.println("----------------------------------------");
+
+        Iterator<String> iterator = words.iterator();
+        while (iterator.hasNext()) {
+            if (iterator.next().length() > 6) {
+                words.remove(iterator.next());
+            } else if (iterator.next().length() < 3) {
+                iterator.next().concat("*");
+            } System.out.println(iterator.next());
+        }
```

Comment:
після перевірки hasNext() треб тільки ращ викликати next().hasNext ми перевіряємо чи є там ще один елемент і якщо є то next() передвигає наш ітератор туда і дає нам те значення. і кількість * повинна бути щоб довжина була 3 цих коротких слів
```suggestion
        while (iterator.hasNext()) {
            String val = iterator.next();
            if (val.length() > 6) {
                iterator.remove();
            } else if (val.length() < 3) {
                iterator.set(val.concat("*".repeat(3 - val.length())));
            }
        }
```